docs/concepts/artifact.mdx

```diff
@@ -64,8 +64,8 @@ flowchart TD
     B -->|No| E[type=manifest.config.mediaType]
 ```
 
-For aritifact authors, the the choice of whether to set the `artifactType` or
-`config.mediaType` depends on the contents and shape of the artifact and is discussed in the following sections.
+For aritifact authors, the sections below should help deciding on the blob and 
```

Comment:
deciding on the blob? Do you mean `artifactType`?

Reply:
Updated and left everything in the basic flowchart. I'm not calling out things like setting annotations since that's been covered in the section above.